packages/apps/tools/src/components/Common/DrawerToolbar/styles.css.ts

```diff
@@ -60,8 +60,8 @@ export const expandedDrawerTitleClass = style([
     display: 'flex',
     justifyContent: 'space-between',
     alignItems: 'center',
-    fontWeight: 'bodyFont.bold',
-    backgroundColor: 'layer-1.default',
+    fontWeight: 'secondaryFont.bold',
+    // backgroundColor: 'layer-1.default',
```

Comment:
What will happen with these commented out styles?

Reply:
It needs backgroundColors (otherwise the sidebar will be transparent)

Reply:
When we implement the new styles we will see if this can be removed.

Reply:
When I wrote this only one comment was visible, I'll check.